resumes/views.py

```diff
@@ -0,0 +1,31 @@
+from django.views       import View
+from django.http        import JsonResponse
+
+from resumes.models     import Resume
+from utils              import authorization
+
+class ResumeView(View):
+    @authorization
+    def get(self, request, resume_id):
+        try:
+            if not Resume.objects.filter(id=resume_id, user=request.user).exists():
+                return JsonResponse({"message" : "USER_NOT_MATCH"}, status=401)
+
+            resume = Resume.objects.get(id=resume_id)
```

Comment:
위와 같은 이유로 get을 할 때 user 정보도 같이 입력해 주셔야 합니다.
```suggestion
            resume = Resume.objects.get(id=resume_id, user=request.user)
```

Reply:
, user=request.user